### PR TITLE
Add Results for Monthly Beatmapping Contest news post

### PR DESCRIPTION
**Pull request by pishifat**
you didn't win the contest but you won my heart ❤️

### CODE REVIEW

**Comment by cl8n** [comment]
i dont feel like ive won cuz u dont reply to my texts

File: news/2020-02-20-results-for-monthly-beatmapping-contest-january.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Votes for all finalists can be found on the [contest's listing](https://osu.ppy.sh/community/contests/84).
+
+**Community vote winner:** [**MYLK - Kiradoki Harmony**](https://osu.ppy.sh/beatmapsets/1111940#osu/2323164) by [**melon boy**](https://osu.ppy.sh/users/3053382) (previously known as Fuccho)
+
+Receiving a towering **78** votes, melon boy's submission climbed to the top of the community vote poll and is crowned the #1 entry in this month's contest! This beatmap features tons of 145 BPM stream jumps with a large circle size, making for a difficult yet manageable alternating-focused experience!
+
+For his reward, melon boy has earned a newly-polished profile badge (design below!) and two months of osu!supporter. See the [**rewards**](#rewards) section for more details.
+
+<center>![MBC winner profile badge: first tier](/wiki/shared/news/2020-02-20-results-for-monthly-beatmapping-contest-january/mbc-badge-1_2x.png)</center>
```
Comment (cl8n) [inline news/2020-02-20-results-for-monthly-beatmapping-contest-january.md]: dont need center tags, it does this automatically on website

File: news/2020-02-20-results-for-monthly-beatmapping-contest-january.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+In case you're out of the loop, the **Monthly Beatmapping Contest** (*MBC*) has been revived for a new run in 2020. Read the contest schedule and details on the [MBC wiki article](/wiki/Contests/Monthly_Beatmapping_Contest). Got everything? Good. Onto the results!
+
+## Winners
+
+Votes for all finalists can be found on the [contest's listing](https://osu.ppy.sh/community/contests/84).
+
+**Community vote winner:** [**MYLK - Kiradoki Harmony**](https://osu.ppy.sh/beatmapsets/1111940#osu/2323164) by [**melon boy**](https://osu.ppy.sh/users/3053382) (previously known as Fuccho)
```
Comment (cl8n) [inline news/2020-02-20-results-for-monthly-beatmapping-contest-january.md]: I think the animal names should be included too so the entries can be recognized

File: news/2020-02-20-results-for-monthly-beatmapping-contest-january.md (hunk 1 above)
Comment (cl8n) [inline news/2020-02-20-results-for-monthly-beatmapping-contest-january.md]: link to the badge directly? dont need to upload it again if it already exists on osu somewhere

File: news/2020-02-20-results-for-monthly-beatmapping-contest-january.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Receiving a towering **78** votes, melon boy's submission climbed to the top of the community vote poll and is crowned the #1 entry in this month's contest! This beatmap features tons of 145 BPM stream jumps with a large circle size, making for a difficult yet manageable alternating-focused experience!
```
Comment (cl8n) [inline news/2020-02-20-results-for-monthly-beatmapping-contest-january.md]: tenses dont match up in first sentence but I thought about this for too long and now I'm not sure what to change to fix it. do u see what i mean at least pls help

File: news/2020-02-20-results-for-monthly-beatmapping-contest-january.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+With an impressive **66** votes, Moecho's entry landed in second place for the contest's community vote! While the users in charge of deciding this contest's finalists were more than exhausted by hearing *Waffle Wave* hundreds of times, they noted that this beatmap's rhythms featuring reversing sliders stood out. If you're looking for a refreshing take on rhythm choice, give this map a shot!
+
+As the contest's runner-up, Moecho earned a month of supporter and is halfway to the first reward tier's profile badge prize! Again, see the [**rewards**](#rewards) section for more details.
+
+
+**Screening crew's favorite:** [**MYLK - Daydreamer**](https://assets.ppy.sh/contests/84/entries/osz/MYLK%20-%20Daydreamer%20(Warm%20Pronghorn).osz?2020) by [**Altai**](https://osu.ppy.sh/users/5745865)
+
+All MBC submissions are screened by about 10 experienced beatmappers to decide the finalists. This month, more than half of the screening crew placed Altai's submission in their top 5 out of over 100 entries! With a high contrast slider velocity gimmick, this beatmap is an experience like no other entry that didn't quite pull in community votes, but may appeal more to the mapper-audience!
```
Comment (cl8n) [inline news/2020-02-20-results-for-monthly-beatmapping-contest-january.md]:
```suggestion
All MBC submissions are screened by about 10 experienced beatmappers to decide the finalists. This month, more than half of the screening crew placed Altai's submission in their top 5 out of over 100 entries! With a high contrast slider velocity gimmick, this beatmap is an experience like no other. It didn't quite pull in community votes, but may appeal more to the mapper audience!
```
confusing qualifier

File: news/2020-02-20-results-for-monthly-beatmapping-contest-january.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+## <a id="rewards"></a>Rewards
+
+As per feedback received on the initial [Monthly Beatmapping Contests Return news post](https://osu.ppy.sh/home/news/2020-01-05-monthly-beatmapping-contests-return), we've adjusted prize thresholds to be more in line with other beatmapping contests! 
+
+MBC points are still distributed to winners: **2** for the community vote winner + **1** for the community vote's runner-up and screening crew's favorite. These are unchanged, however the threshold for rewards have been adjusted:
+
+- **Profile badge:** badges are attainable at 2, 4, and 6 points each. The badges are tiered, meaning each new badge will replace the previous one (and they get progressively shinier!).
+- **"Elite Mapper" forum title:** attainable at 6 points.
+- **osu!supporter:** each point earned provides a user with a month of supporter for themself or someone of their choosing.
```
Comment (cl8n) [inline news/2020-02-20-results-for-monthly-beatmapping-contest-january.md]:
```suggestion
- **osu!supporter:** each point earned provides a user with a month of supporter for themselves or someone of their choosing.
```

File: news/2020-02-20-results-for-monthly-beatmapping-contest-january.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+A points breakdowns for all MBC participants can be found on the [MBC wiki article](/wiki/Contests/Monthly_Beatmapping_Contest#points).
```
Comment (cl8n) [inline news/2020-02-20-results-for-monthly-beatmapping-contest-january.md]:
```suggestion
A points breakdown for all MBC participants can be found on the [MBC wiki article](/wiki/Contests/Monthly_Beatmapping_Contest#points). 
```

File: news/2020-02-20-results-for-monthly-beatmapping-contest-january.md (hunk 1 above)
Comment (Ephemeralis) [reply news/2020-02-20-results-for-monthly-beatmapping-contest-january.md]: it doesn't as of the time of writing, no real harm in including it from the article

**Pull request merged**